src/components/thumbnail/collapsible-document-section.tsx

```diff
@@ -1,22 +1,25 @@
 import React, { useState } from "react";
 import { observer } from "mobx-react";
+import { DocumentContextReact } from "../document/document-context";
 import { INetworkResourceClassResponse } from "../../../functions/src/shared";
-import { DocumentModelType } from "../../models/document/document";
-// import { DocumentCaption } from "./document-caption";
+import { DocumentModelType, getDocumentContext } from "../../models/document/document";
 import { IStores } from "../../models/stores/stores";
 import ArrowIcon from "../../assets/icons/arrow/arrow.svg";
-// import NotSharedIcon from "../../assets/icons/share/not-share.svg";
 import { ISubTabSpec } from "../navigation/document-tab-panel";
+import { useNetworkDocuments } from "../../hooks/use-stores";
+import { TabPanelDocumentsSubSectionPanel } from "./tab-panel-documents-subsection-panel";
+import { NavTabSectionModelType } from "../../models/view/nav-tabs";
+// import NotSharedIcon from "../../assets/icons/share/not-share.svg";
```

Comment:
Not for this PR, but just noting that problem and planning documents have a `visibility` field that can be used to determine whether they have been shared. I believe the spec indicates that teachers control whether the contents of those documents are available over the network, and there's a different icon for unshared content.